styles.css

```diff
@@ -0,0 +1,16 @@
+form {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+input, select, textarea{
+  margin: 10px auto;
+  padding: 7px 0;
+  width: 100%;
+}
+
+checkboxes, .radio-buttons{
+  display: flex;
+  width: 20%;
+  
+}
```

Comment:
Please run the style linters in your stylesheet to fix any errors. To automatically fix, add *--fix* to the command provided in the [linters repo](https://github.com/microverseinc/linters-config/tree/master/html-css)😊